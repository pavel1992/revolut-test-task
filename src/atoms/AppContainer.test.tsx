import * as React from 'react';
import renderer from 'react-test-renderer';

import { AppContainer } from "./AppÐ¡ontainer";


test('renders app container correctly', () => {
  const renderedApp = renderer
    .create(<AppContainer/>)
    .toJSON();
  expect(renderedApp).toMatchSnapshot();
});
