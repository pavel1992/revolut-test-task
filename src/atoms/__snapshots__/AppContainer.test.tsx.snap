// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app container correctly 1`] = `
<div
  className="sc-AxjAm cznkoO"
/>
`;
